### PR TITLE
v1.3.1 Updates

### PR DESCRIPTION
**Pull request by faberga**
v1.3.1 Updates

### CODE REVIEW

**Comment by faberga** [comment]
Jenkins run tests

File: CHANGES.md [hunk 1]
```diff
@@ -1,3 +1,12 @@
+HElib 1.3.1, December 2020
+=========================
+(tagged as v1.3.1)
+
+* Refactoring of CKKS Mitigation
+* Re-instated ctxt isCorrect
```
Comment (esteffin) [inline CHANGES.md:6]:
```suggestion
* Re-instated `Ctxt` isCorrect
```

File: CHANGES.md [hunk 2]
```diff
@@ -1,3 +1,12 @@
...
+* Conjugation fix
```
Comment (esteffin) [inline CHANGES.md:7]:
```suggestion
* Fixed `Ctxt` conjugation
```

**Review by esteffin — APPROVED** [review]
Good for me.

File: CHANGES.md (hunk 2 above)
Comment (faberga) [inline CHANGES.md:7]:
```suggestion
* Ctxt Conjugation fix
```

**Pull request merged**